ledger-core/virtual/integration/method_test.go

```diff
@@ -1326,8 +1325,7 @@ func Test_MethodCall_HappyPath(t *testing.T) {
 		server.IncrementPulseAndWaitIdle(ctx)
 		outgoing := server.BuildRandomOutgoingWithPulse()
 
-		// add ExecutionMock to runnerMock
```

Comment:
I think it's better because you don't have to open a block to understand what it's about.

Reply:
return it on separate line